source/server/server.cc

```diff
@@ -1009,9 +1009,6 @@ bool InstanceImpl::enableReusePortDefault() {
     return true;
   case ReusePortDefault::False:
     return false;
-  case ReusePortDefault::Runtime:
-    return Runtime::runtimeFeatureEnabled(
-        "envoy.reloadable_features.listener_reuse_port_default_enabled");
```

Comment:
Is that means we only have `true` and `false` value for the `enable_reuse_port_default_`? so `enable_reuse_port_default_` can be a normal bool instead of the enum `ReusePortDefault`, and the enum `ReusePortDefault` becomes useless.

Reply:
> Is that means we only have `true` and `false` value for the `enable_reuse_port_default_`? so `enable_reuse_port_default_` can be a normal bool instead of the enum `ReusePortDefault`, and the enum `ReusePortDefault` becomes useless.

Yeah, I think it's good to remove the enum `ReusePortDefault`.